typescript/infra/src/config/agent.ts

```diff
@@ -130,11 +132,16 @@ type ChainRelayerConfigs<Chain extends ChainName> = ChainOverridableConfig<
 
 // Full relayer agent config for a single chain
 interface RelayerConfig
-  extends Omit<BaseRelayerConfig, 'whitelist' | 'blacklist'> {
+  extends Omit<
+    BaseRelayerConfig,
+    'whitelist' | 'blacklist' | 'skipTransactionGasLimitFor'
+  > {
   originChainName: ChainName;
   multisigCheckpointSyncer: MultisigCheckpointSyncerConfig;
   whitelist?: string;
   blacklist?: string;
+  transactionGasLimit?: string;
```

Comment:
I *think* if this we keep `transactionGasLimit` as a string in BaseRelayerConfig, then we don't need to redeclare it here